docs/installation.md

```diff
@@ -283,32 +283,14 @@ sudo dnf install podman
 
 **Required**
 
-Fedora, CentOS, RHEL, and related distributions you should try to run
-`make package-install` which will install dependencies, build the source,
+On Fedora, CentOS Stream, RHEL, and related distributions you should run
+the following steps which will install dependencies, build the source,
 produce rpms for the current platform and install them in the end.
 
 ```bash
-sudo yum install -y \
-  btrfs-progs-devel \
-  conmon \
-  containernetworking-plugins \
-  containers-common \
-  crun \
-  device-mapper-devel \
-  git \
-  glib2-devel \
-  glibc-devel \
-  glibc-static \
-  go \
-  golang-github-cpuguy83-md2man \
-  gpgme-devel \
-  iptables \
-  libassuan-devel \
-  libgpg-error-devel \
-  libseccomp-devel \
-  libselinux-devel \
-  make \
-  pkgconfig
+sudo dnf -y builddep rpm/podman.spec
```

Comment:
will this only install build deps?
What about the runtime deps such as conmon, crun, netavark, etc...? Or will the rpm-install step pull these in?

Reply:
yes, builddep only installs build dependencies. The third step `sudo make rpm-install` will do a `dnf install $RPM_PATH` which will install the runtime deps.

Reply:
I can add comments in there if it helps

Reply:
comments would be appreciated